crates/nu-plugin-core/src/serializers/tests.rs

```diff
@@ -341,7 +362,10 @@ macro_rules! generate_tests {
             match returned {
                 PluginOutput::CallResponse(
                     5,
-                    PluginCallResponse::PipelineData(PipelineDataHeader::Value(returned_value)),
+                    PluginCallResponse::PipelineData(PipelineDataHeader::Value {
```

Comment:
You might consider adding a helper like `PipelineDataHeader::value()` for no metadata, since it seems to be quite common

Reply:
I'll add it!